cg/server/api.py

```diff
@@ -143,15 +143,15 @@ def submit_order(order_type):
 
 
 @BLUEPRINT.route("/cases")
-def cases():
+def render_cases():
```

Comment:
I'm not sure about introducing the `render` term. Do you oppose just `get`?

Reply:
getting is done by the getters within those functions. maybe `get_to_render` ?

Reply:
Hmm, maybe parse_ since the return is a json representation of the data. Rendering is done in the HTML I believe so it is not the right naming indeed...

what about `parse_`

Reply:
I'm not sure what the best practice is for naming the functions used as endpoints. I have no issue with just calling it get_cases. @seallard do you have an opinon here?